README.md

```diff
@@ -105,28 +105,43 @@ file with the appropriate information for accessing it.
 2. In the `.env` file, ensure that the following have been set correctly to access your local database:
    * `DB_HOST`: usually `localhost` or `127.0.0.1`
    * `DB_PORT`: usually `3306`
-   * `DB_DATABASE`: will likely need to create a new database in MySQL or MariaDB first
+   * `DB_DATABASE`: usually `licl`; will likely need to create a new database in MySQL or MariaDB first
    * `DB_USERNAME`
    * `DB_PASSWORD`
 
-3. Ensure that a database called `lcil_test` is also created in your local database. This is used for running the tests.
+3. If you need to create a database you can do so from the command line like:
+
+    ```bash
+    mysql -uroot -e "create database lcil;"
+    ```
+
+    You can also create the database from an application like [Sequel Ace](https://github.com/Sequel-Ace/Sequel-Ace).
+
+    _**NOTE:** If the database is run through an external app like [DBngin](https://dbngin.com) the CLI command above
+    may not work._
```

Comment:
@jhung I added some DB creation info to the README. It might be a bit different depending on your installation and platform. But hopefully this will help.